src/Services.Remoting/Client/OmexServiceProxyFactory.cs

```diff
@@ -30,6 +30,7 @@ public static class OmexServiceProxyFactory
 		/// </summary>
 		public static void WithCustomTransportSettings(FabricTransportRemotingSettings transportSettings)
 		{
+			transportSettings.ExceptionDeserializationTechnique = FabricTransportRemotingSettings.ExceptionDeserialization.Default;
```

Comment:
s_serviceProxyFactory initialization above needs to be updated with transport settings setting FabricTransportRemotingSettings.ExceptionDeserialization.Default too.

Reply:
yes, added it now!